### ISSUE TITLE
Feat Request: Add SSO auth for thelounge

### ISSUE THREAD

**Issue by ben-burns** [saltyorg/Sandbox#234]
I specifically avoid using thelounge on my saltbox setup because I don't want thelounge exposed to the public internet without another layer of auth in front of it (e.g., SSO).  It would be nice to have Authelia in front of thelounge before you can log in (much like one can do with the *arrs).

**Comment by ben-burns** [comment]
Thanks for adding this!  I've noticed after updating `sb` and thelounge image, it appears to not hit Authelia (also confirmed from Authelia logs).  I've checked that the role has been updated to what is in https://github.com/saltyorg/Sandbox/commit/0bccfc2a432c9a2d8fbf8064b53c32d329d683c2, as well as reverted and tried ```rolename_traefik_middleware_custom: "authelia"``` to no avail.  

I'll have some more time in the coming day to investigate further, however, I wanted to reach out in case there is something simple I have missed.
